assets/timetable.csv

```diff
@@ -1,12 +1,12 @@
-9c45718b-55e4-4130-a590-0c7fb72ceef4,,dorcester-new-local-plan,publication-start,2024-02,,,2024-02-02T15:46:14.144Z,2024-02-02T15:46:14.144Z,
-70ce45bb-509e-4749-9d61-4a05315bc7e1,,dorcester-new-local-plan,publication-end,2024-02,,,2024-02-02T15:46:14.144Z,2024-02-02T15:46:14.144Z,
-d26178c6-8c07-448e-890b-b4332ec0dcae,,dorcester-new-local-plan,plan-submitted-for-examination,2024-02,,,2024-02-02T15:46:14.144Z,2024-02-02T15:46:14.144Z,
-f602ce0b-1abd-491f-bb58-46ec8feb7c0f,,dorcester-new-local-plan,examination-hearing-start,2024-02,,,2024-02-02T15:46:14.144Z,2024-02-02T15:46:14.144Z,
-9d238d4d-4635-4f82-8a60-ccfdab0b2c99,,dorcester-new-local-plan,examination-hearing-end,2024-02,,,2024-02-02T15:46:14.144Z,2024-02-02T15:46:14.144Z,
-b85f4dbc-30af-4d30-943b-c0c103b296c8,,dorcester-new-local-plan,plan-adopted,2024-02,,,2024-02-02T15:46:14.144Z,2024-02-02T15:46:14.144Z,
-beb5847a-f802-43d7-ac69-b5d6a567b9a0,,dorcester-new-local-plan,local-development-scheme-published,2024-02,,,2024-02-02T15:46:14.144Z,2024-02-02T15:46:14.144Z,
\ No newline at end of file
+39b7f5e2-beba-43d7-b20d-d9ca3b710f80,,dorcester-new-local-plan,timetable-updated,2024-02,,,2024-02-02T15:46:14.144Z,2024-02-02T15:46:14.144Z,2024-02-02T15:46:14.144Z,
+7336edfb-8db3-4bbc-806f-4a0895cdeea6,,dorcester-new-local-plan,local-development-scheme-published,2024-02,,,2024-02-02T15:46:14.144Z,2024-02-02T15:46:14.144Z,2024-02-02T15:46:14.144Z,
+267645e9-7643-4d00-a33c-cebc39e898ef,,dorcester-new-local-plan,public-consultation-start,2024-02,,,2024-02-02T15:46:14.144Z,2024-02-02T15:46:14.144Z,2024-02-02T15:46:14.144Z,
+2968bdd3-6ea0-4129-b3ad-10980bfb396a,,dorcester-new-local-plan,public-consultation-end,2024-02,,,2024-02-02T15:46:14.144Z,2024-02-02T15:46:14.144Z,2024-02-02T15:46:14.144Z,
+9c45718b-55e4-4130-a590-0c7fb72ceef4,,dorcester-new-local-plan,publication-start,2024-02,,,2024-02-02T15:46:14.144Z,2024-02-02T15:46:14.144Z,2024-02-02T15:46:14.144Z,
+70ce45bb-509e-4749-9d61-4a05315bc7e1,,dorcester-new-local-plan,publication-end,2024-02,,,2024-02-02T15:46:14.144Z,2024-02-02T15:46:14.144Z,2024-02-02T15:46:14.144Z,
+d26178c6-8c07-448e-890b-b4332ec0dcae,,dorcester-new-local-plan,plan-submitted-for-examination,2024-02,,,2024-02-02T15:46:14.144Z,2024-02-02T15:46:14.144Z,2024-02-02T15:46:14.144Z,
+f602ce0b-1abd-491f-bb58-46ec8feb7c0f,,dorcester-new-local-plan,examination-hearing-start,2024-02,,,2024-02-02T15:46:14.144Z,2024-02-02T15:46:14.144Z,2024-02-02T15:46:14.144Z,
+9d238d4d-4635-4f82-8a60-ccfdab0b2c99,,dorcester-new-local-plan,examination-hearing-end,2024-02,,,2024-02-02T15:46:14.144Z,2024-02-02T15:46:14.144Z,2024-02-02T15:46:14.144Z,
+b85f4dbc-30af-4d30-943b-c0c103b296c8,,dorcester-new-local-plan,plan-adopted,2024-02,,,2024-02-02T15:46:14.144Z,2024-02-02T15:46:14.144Z,2024-02-02T15:46:14.144Z,
+beb5847a-f802-43d7-ac69-b5d6a567b9a0,,dorcester-new-local-plan,local-development-scheme-published,2024-02,,,2024-02-02T15:46:14.144Z,2024-02-02T15:46:14.144Z,2024-02-02T15:46:14.144Z,
```

Comment:
This isn't right, we have no valid events?

Reply:
my bad. I thought this was causing weirdness with the CSV function and I forgot to revert. Its done now :)